schema/email-preferences.js

```diff
@@ -0,0 +1,42 @@
+const { Model } = require('objection');
+const { uuid } = require('../lib/regex-validation');
+
+class EmailPreferences extends Model {
+  static get tableName() {
+    return 'emailPreferences';
+  }
+
+  $beforeUpdate() {
+    this.updatedAt = new Date().toISOString();
```

Comment:
how come we aren't extending the base model where this is default behaviour?

Reply:
Using BaseModel requires having a `deleted` column which will never be used - once a user saves prefs they have them forever, we just modify the existing record.

Reply:
OK cool thanks 👍